src/Illuminate/Support/Debug/Dumper.php

```diff
@@ -15,6 +15,12 @@ class Dumper
      */
     public function dump($value)
     {
+        if (! class_exists(CliDumper::class)) {
+            var_dump($value);
+
+            return;
```

Comment:
the return type of this method is void, so just write `return var_dump($value)`.